pilot/pkg/security/authn/factory/factory.go

```diff
@@ -17,17 +17,17 @@ package factory
 import (
 	"istio.io/istio/pilot/pkg/model"
 	"istio.io/istio/pilot/pkg/security/authn"
-	"istio.io/istio/pilot/pkg/security/authn/v1alpha1"
+	"istio.io/istio/pilot/pkg/security/authn/v1beta1"
+	"istio.io/istio/pkg/config/labels"
 )
 
 // NewPolicyApplier returns the appropriate (policy) applier, depends on the versions of the policy exists
 // for the given service instance.
 func NewPolicyApplier(push *model.PushContext,
-	serviceInstance *model.ServiceInstance) authn.PolicyApplier {
-	// TODO: check v1alpha2 policy and returns alpha2 applier, if exists.
+	serviceInstance *model.ServiceInstance, namespace string, labels labels.Collection) authn.PolicyApplier {
```

Comment:
May be pass the Node (proxy) instead? I think it can replace ServiceInstance at some point

Reply:
resolving per offline discussoin.